### PR TITLE
Add ability to specify listen variables through the environment

### PR DESCRIPTION
**Pull request by MatthewWilkes**
Python 3.8 has a `breakpoint()` built in to start the specified debugger. This change allows you to specify host and port details in the environment, so software that uses the `breakpoint()` debugging convention can be run with remote-pdb without needing specific code changes and without having access to stdout.

This helps with simplifying the learning journey for users on 3.8 when debugging complex multi-threaded or multi-process systems.

As with the other PR, *please* do ask if there's anything you want me to change or do differently.

Matt

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -93,6 +93,20 @@ Alternately, one can connect with NetCat: ``nc -C 127.0.0.1 4444`` or Socat: ``s
...
+
+If you are using Python 3.8 you can specify that ``breakpoint()`` should invoke
+remote_pdb. In this case, you may use environment variables to specify the
+host and port that should be used:
+
+.. code:: bash
+
+    PYTHONBREAKPOINT=remote_pdb.set_trace REMOTE_PDB_HOST=0.0.0.0 REMOTE_PDB_PORT=10000 python script.py
+
+Omitting either of these environment variables will use the default logic from above.
```
Comment (ionelmc) [inline README.rst]: Hmmm. I'd reword this somehow, it doesn't sound right. Perhaps just state what the defaults are?

Comment (MatthewWilkes) [reply README.rst]: I've reworked the whole section to try and be more explicit and be closer to the voice you used in the other parts of the readme. I'd originally tried to make this a light touch section to avoid restating things, but your idea of being explicit works for me.

**Comment by ionelmc** [comment]
Very good. Thanks.
